misc/drone-scripts/setup.sh

```diff
@@ -19,22 +19,29 @@ govc snapshot.revert -vm $ESX_6_0 init
 
 echo "Waiting for revert to complete";
 
+DIR=$(dirname ${BASH_SOURCE[0]})
+. $DIR/../scripts/wait_for.sh
+
+# Threshold to time out 
```

Comment:
wait_for method check every 1 seconds and threshold has been set for 30 time i.e. 30 seconds maximum to time out.